NEWS.md

```diff
@@ -4,6 +4,11 @@
 - Added claude-3.5-sonnet model from Anthropic.
 - Set gpt-4o-mini as default model for OpenAI. #219
 - Fixed bugs with Azure OpenAI service. #223
+- Add audio input option for chat app. #224
+- Fix bug with chat app not loading on linux. #224
+- Allow chat app to run in Positron (not yet as background job) #224
+- API calls now run async with ExtendedTask. #224
```

Comment:
I don't think async (and by extension, `{promises}`) is really needed here, since the addin is supposed to work for a single machine. I'm ok with ExtendedTask with the new waiting button and loaders tho (in this case we need to add a minimun version to shiny in DESCRIPTION)

Reply:
This was meant for line 10

Reply:
Fair point on async. I've been using local models more, and they can be _slow_. The idea was to get not block actions in the current session, but I don't think that adds much. I'll remove the `future_promise()` call from the `ExtendedTask`. Easy to go back to it if we want later.

Reply:
Update on this, ExtendedTask needs a promise to be returned. I've moved promises to Suggests, but it's not really another dependency because shiny already depends on the package.